Python/getargs.c

```diff
@@ -848,8 +848,18 @@ convertsimple(PyObject *arg, const char **p_format, va_list *p_va, int flags,
     }
 
     case 'p': {/* boolean *p*redicate */
-        int *p = va_arg(*p_va, int *);
         int val = PyObject_IsTrue(arg);
+        if (*format == '*') {
+            // p*
```

Comment:
```suggestion
```

Reply:
I would rather keep this one for consistency, some of the other format characters have a comment like that.